Podfile.lock

```diff
@@ -154,8 +154,8 @@ SPEC CHECKSUMS:
   ARAnalytics: 5468652928cee7100dd0e6b5162631409da37106
   ARCollectionViewMasonryLayout: 164b82010cf8ec99bc7a38cffe59a179d7e5a116
   ARTiledImageView: a747cff42142ca04d1dc5cee516186f10b6c0949
+  Artsy+OSSUIFonts: e538d656b33ee7b1c66a97f57923f37ce565b3a8
   Artsy+UIColors: 7a4c987885a8d8da3e22672a3232761f929dd2b6
-  Artsy+UIFonts: c51bb3b5cbf9c1a5fe198b4385d49133c5c71f5a
```

Comment:
Ah, it's a command run from the directory root. It was invoked by fastlane originally (you likely have stubbed API keys). We need to improve our non-OSS, Artsy-team setup docs.